lib/cocoapods/user_interface.rb

```diff
@@ -19,6 +19,7 @@ class << self
       attr_accessor :indentation_level
       attr_accessor :title_level
       attr_accessor :warnings
+      attr_accessor :io
```

Comment:
rename this to `output` and I think we're good to go, unless @Karla-Isabel-Sandoval wants to give any feedback